src/components/Images/Images.md

```diff
@@ -0,0 +1,81 @@
+
+ ## Cloudinary crop:
+  https://cloudinary.com/documentation/image_transformation_reference#crop_parameter
+
+* fill: Create an image with the exact given width and height without distorting the
+      image. This option first scales as much as needed to at least fill both of the
+      given dimensions. If the requested aspect ratio is different than the original,
+      cropping will occur on the dimension that exceeds the requested size after scaling.
+      Similar to CSS background-size "cover".
+
+* fit: The image is resized so that it takes up as much space as possible within
+      a bounding box defined by the given width and height parameters. The original
+      aspect ratio is retained and all of the original image is visible.
+      Similar to CSS background-size "contain".
+
+* crop: Used to extract a given width & height out of the original image. The original
+      proportions are retained and so is the size of the graphics.
+      Similar to CSS background-size "auto".
+
+```jsx
+import {
+  TitleSmall,
+  TextInput,
+  Spacer,
+  Button,
+  InfoMessage,
+  ZipInput,
+  EmailInput,
+} from '../index'
+
+;<div style={{ width: 400 }}>
+  <TitleSmall.Serif.Book500>Minimal Props</TitleSmall.Serif.Book500>
+  <Spacer.H24/>
+  <CloudinaryImage
+    alt="father and kids playing"
+    publicId="https://res.cloudinary.com/getethos/image/upload/v1565712179/01_NEW%20Lifestyle%20%28Rebrand%29/life-insurance-father-and-kids-playing.jpg"
```

Comment:
How do we know these images won't go away?

Reply:
Yeah these could be removed anytime, what would be an appropriate image to use?

Reply:
because our image elements are baking in cloudinary dependence i don't see any other way to do this.